src/bot.coffee

```diff
@@ -199,4 +204,32 @@ class SlackBot extends Adapter
     item_user = @client.rtm.dataStore.getUserById(item_user)
     @receive new ReactionMessage(type, user, reaction, item_user, item, event_ts)
 
+  loadUsers: (err, res) =>
+    if err || !res.ok
+      @robot.logger.error "Can't fetch users"
+      return
+
+    for id, user of res.members
+      @userChange user
+
+  userChange: (user) =>
+    return unless user?.id?
+    newUser =
+      name: user.name
```

Comment:
can we store `id: user.id` in here as well? if the idea that Slack-specific properties should go inside the `slack` property, then this is already not working since the ID that the user will be identified with in the brain is the Slack ID.